package.json

```diff
@@ -10,8 +10,7 @@
     "@typescript-eslint/eslint-plugin": "2.14.0",
     "@typescript-eslint/parser": "2.14.0",
     "babel-eslint": "10.0.3",
-    "babel-jest": "24.9.0",
-    "eslint": "5.16.0",
+    "eslint": "6.8.0",
```

Comment:
ESLint does need to be installed at this level, the config can't resolve things it needs without it. However, moving this to a higher version doesn't break subrepos that haven't been updated so it looks like the final source of truth is the version of ESLint that initiates the run.